recipe/meta.yaml

```diff
@@ -1,48 +1,32 @@
-{% set gh_org = "jupyter" %}
-{% set gh_repo = "nbgrader" %}
 {% set version = "v0.5.1" %}
-{% set sha256 = "971a37b58a8f1a26e7b17eacb091879b5f1646f5097b717626436a266a178d33" %}
 
 package:
-  name: {{ gh_repo|lower }}
```

Comment:
I removed the `jinja` excess to make the recipe more readable. We only want jinja variables for things that change every update and in more than one place.